### PR TITLE
Increase Scylla probes timeout to 30s

### PR DESCRIPTION
**Pull request by zimnx**
Related to #844

### CODE REVIEW

**Review by tnozicka — COMMENTED** [review]
let's raise startup probe as well

**Comment by zimnx** [comment]
> let's raise startup probe as well

done

File: pkg/controller/scyllacluster/resource/resource.go [hunk 1]
```diff
@@ -342,7 +344,9 @@ func StatefulSetForRack(r scyllav1.RackSpec, c *scyllav1.ScyllaCluster, existing
 								},
 							},
 							LivenessProbe: &corev1.Probe{
-								TimeoutSeconds:   int32(5),
+								// TODO: temporarily increase timeout from 5s to 30s to survive node overload
```
Comment (tnozicka) [inline pkg/controller/scyllacluster/resource/resource.go:347]: nit: the TODO should say what needs to be done in the future. Like `TODO: Lower the timeout when we fix probe timeouts. We have temporarily increased the timeout not to kill nodes when under load.` 

The issue that is reverenced should have a clear action item to remove this `TODO` and lower the timeout.

**Review by tnozicka — APPROVED** [review]
/lgmt

**Review by tnozicka — APPROVED** [review]

**Pull request merged**